vuepress/docs/next/docs/reference/databases.md

```diff
@@ -294,52 +310,53 @@ To achieve this, specify the pertinent properties for the EntandoApp component i
-          - name: SPRING_DATASOURCE_URL
-            value: "jdbc:postgresql://192.168.1.82:5432/testdb?currentSchema=admin_qs_dedb"
-          - name: SPRING_JPA_DATABASE_PLATFORM
-            value: org.hibernate.dialect.PostgreSQLDialect
-          - name: PORTDB_URL
-            value: "jdbc:postgresql://192.168.1.82:5432/testdb?currentSchema=admin_qs_portdb"
-          - name: PORTDB_USERNAME
-            value: admin
-          - name: PORTDB_PASSWORD
-            value: adminadmin
-          - name: PORTDB_CONNECTION_CHECKER
-            value: org.jboss.jca.adapters.jdbc.extensions.postgres.PostgreSQLValidConnectionChecker
-          - name: PORTDB_EXCEPTION_SORTER
-            value: org.jboss.jca.adapters.jdbc.extensions.postgres.PostgreSQLExceptionSorter
-          - name: SERVDB_URL
-            value: "jdbc:postgresql://192.168.1.82:5432/testdb?currentSchema=admin_qs_servdb"
-          - name: SERVDB_USERNAME
-            value: admin
-          - name: SERVDB_PASSWORD
-            value: adminadmin
-          - name: SERVDB_CONNECTION_CHECKER
-            value: org.jboss.jca.adapters.jdbc.extensions.postgres.PostgreSQLValidConnectionChecker
-          - name: SERVDB_EXCEPTION_SORTER
-            value: org.jboss.jca.adapters.jdbc.extensions.postgres.PostgreSQLExceptionSorter
+   kind: "EntandoApp"
+metadata:
+  name: "my-app"
+spec:
+  dbms: "none"
+  replicas: 1
+  ingressHostName: "my-app.192.168.1.100.nip.io"
+  standardServerImage: "eap"
+  environmentVariables:
+    - name: SPRING_DATASOURCE_USERNAME
+      value: admin
+    - name: SPRING_DATASOURCE_PASSWORD
+      value: adminadmin
+    - name: SPRING_DATASOURCE_URL
+      value: "jdbc:postgresql://192.168.1.82:5432/testdb?currentSchema=admin_qs_dedb"
+    - name: SPRING_JPA_DATABASE_PLATFORM
+      value: org.hibernate.dialect.PostgreSQLDialect
+    - name: PORTDB_URL
+      value: "jdbc:postgresql://192.168.1.82:5432/testdb?currentSchema=admin_qs_portdb"
+    - name: PORTDB_USERNAME
+      value: admin
+    - name: PORTDB_PASSWORD
+      value: adminadmin
+    - name: PORTDB_CONNECTION_CHECKER
+      value: org.jboss.jca.adapters.jdbc.extensions.postgres.PostgreSQLValidConnectionChecker
+    - name: PORTDB_EXCEPTION_SORTER
+      value: org.jboss.jca.adapters.jdbc.extensions.postgres.PostgreSQLExceptionSorter
+    - name: SERVDB_URL
+      value: "jdbc:postgresql://192.168.1.82:5432/testdb?currentSchema=admin_qs_servdb"
+    - name: SERVDB_USERNAME
+      value: admin
+    - name: SERVDB_PASSWORD
+      value: adminadmin
+    - name: SERVDB_CONNECTION_CHECKER
+      value: org.jboss.jca.adapters.jdbc.extensions.postgres.PostgreSQLValidConnectionChecker
+    - name: SERVDB_EXCEPTION_SORTER
+      value: org.jboss.jca.adapters.jdbc.extensions.postgres.PostgreSQLExceptionSorter
 ```
+**Note**: This configuration is not meant to be used as a template for a production environment.
+Use K8s Secrets in such cases to store the credentials and the syntax indicated here.
```

Comment:
sentence is a little jumbled wrt actions associated with the verb "use" and "in such cases" can be avoided for a more direct and succinct instruction: "instead (as in for a production environment), store the credentials via K8s Secrets and use the syntax indicated here."

Reply:
I don't think the instead works here. The first sentence is a general recommendation, and the second is one of the a specific details of note. I did edit it with part of the rec

Reply:
i misread it then - it took "in such cases" to mean that the second sentence was an extension of the first statement